Makefile

```diff
@@ -132,15 +132,18 @@ init-golang-rly: kill-dev install
 	./network/init.sh
 	./network/start.sh
 	@echo "Initializing relayer..."
-	./network/relayer/interchain-acc-config/rly.sh
+	./network/relayer/interchain-acc-config/rly-init.sh
 
 start: 
 	@echo "Starting up test network"
 	./network/start.sh
 
-start-rly:
+start-hermes:
 	./network/hermes/start.sh
 
+start-golang-rly:
```

Comment:
We could maybe leave this as `start-rly` for brevity, thoughts? 
I'm happy with either

Reply:
Yeah, that would be fine by me. So have `start-hermes` and `start-rly` (and same for init). Looks cleaner!